src/packages/calendar/calendar.taro.tsx

```
@@ -87,7 +87,7 @@ export const Calendar = React.forwardRef<
     close()
   }
 
-  const select = (param: string) => {
+  const select = (param: string[]) => {
     onDayClick && onDayClick(param)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)